src/ipc/bridge.cc

```diff
@@ -2601,7 +2612,14 @@ static void registerSchemeHandler (Router *router) {
     body = (char *) data;
   }
 
+  [_tasks addObject: task];
+
   auto invoked = self.router->invoke(message, body, bufsize, [=](Result result) {
+    // @TODO Communicate task cancellation to the route, so it can cancel its work.
```

Comment:
In the future, we'll want a way for route invocations to register a callback for task cancellation.